src/trix/views/document_view.coffee

```diff
@@ -23,7 +23,7 @@ class Trix.DocumentView extends Trix.ObjectView
   render: ->
     @childViews = []
 
-    @shadowElement = makeElement("div")
+    @element.shadowElement = @shadowElement = makeElement("div")
```

Comment:
The editor element should probably create this element when constructed if it's going to be a permanent property. Setting it here is too far removed. Generally, moving serialization out of the element and into EditorController might help centralize the work: https://github.com/basecamp/trix/blob/d4b81b80e23c1a1ef08486f1ec9982808ec66fa6/src/trix/elements/trix_editor_element.coffee#L156-L157